DCCpp_Uno/src/PacketRegister.cpp

```diff
@@ -225,9 +225,10 @@ void RegisterList::readCV(const char *s) volatile{
 
     bRead[2]=0xE8+i;
 
-    loadPacket(0,resetPacket,2,3);          // NMRA recommends starting with 3 reset packets
-    loadPacket(0,bRead,3,5);                // NMRA recommends 5 verfy packets
-    loadPacket(0,resetPacket,2,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
+loadPacket(0,resetPacket,2,3); // NMRA recommends starting with 3 reset packets
+loadPacket(0,bRead,3,5); // NMRA recommends 5 verify packets
+loadPacket(0,bRead,3,6); // NMRA recommends 6 write or reset packets for decoder recovery time
+loadPacket(0,resetPacket,2,1); // Final reset packet completed (and decoder begins to respond)
```

Comment:
move the reset packet until after the ACK_SAMPLE_COUNT loop below, the last step of a CV read/write operation should be send a reset packet to the decoder.